src/app/containers/StoryPromo/LinkContents/index.jsx

```diff
@@ -35,27 +35,37 @@ const LinkContents = ({ item }) => {
 
   // Always gets the first version. Smarter logic may be needed in the future.
   const rawDuration = pathOr(null, ['media', 'versions', 0, 'duration'], item);
+  let offScreenDuration;
 
-  const separator = ',';
-  const duration = moment.duration(rawDuration, 'seconds');
-  const durationString = formatDuration({ duration, separator });
+  if (rawDuration && !isInline) {
+    const separator = ',';
+    const duration = moment.duration(rawDuration, 'seconds');
+    const durationString = formatDuration({ duration, separator });
+
+    offScreenDuration = (
+      // once we have 'duration' translations, we could place those here
+      <VisuallyHiddenText>{`, ${durationString}`}</VisuallyHiddenText>
```

Comment:
```suggestion
      <VisuallyHiddenText>{`${separator} ${durationString}`}</VisuallyHiddenText>
```

Reply:
Not sure about this, because we always want a comma before the duration, but the separator in the duration could change in the future.

Reply:
Okay, I was thinking that the separator in the visuallyhiddentext string should be localised. We'd need to do that for all of our visually hidden text across the application, so we could handle this in future.